bin/alluxio-start.sh

```diff
@@ -515,11 +515,26 @@ main() {
       start_proxies
       ;;
     local)
-      ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.hostname=localhost"
-      ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.master.hostname=localhost"
-      ALLUXIO_PROXY_JAVA_OPTS+=" -Dalluxio.master.hostname=localhost"
-      ALLUXIO_JOB_MASTER_JAVA_OPTS+=" -Dalluxio.master.hostname=localhost"
-      ALLUXIO_JOB_WORKER_JAVA_OPTS+=" -Dalluxio.master.hostname=localhost"
+      local master_hostname=$(${BIN}/alluxio getConf ${ALLUXIO_MASTER_JAVA_OPTS}\
+                              alluxio.master.hostname)
+      if [[ -n ${master_hostname} ]]; then
```

Comment:
Do we need to do anything if the master hostname is set?

Reply:
good catch, fixed